end-to-end-test/test_end_to_end.py

```diff
@@ -217,6 +217,7 @@ def test_cog_safe_push_train():
                     train_destination_name=test_model_name + "-dest",
                 ),
                 fuzz_iterations=1,
```

Comment:
I wasn't entirely sure what this test was trying to do with testing the second training in a row. Outputs aren't going to be equivalent b/c the `weights` output has some hash we use for naming the weights bucket - if there's another way to reconfigure this test to capture the original intent I'm all for it.

Reply:
I think the second test is to make sure the schema hasn't changed between trainings.

Reply:
ah! makes sense.